realsense/plugin.cpp

```diff
@@ -122,23 +243,27 @@ class realsense : public plugin {
                         rgb = cam_type_.rgb;
                         depth = cam_type_.depth;
                     }
-
+                    
                     // Submit to switchboard
                     _m_imu_cam.put(_m_imu_cam.allocate<imu_cam_type>(
-                        imu_time_point,
-                        av,
-                        la,
-                        img0,
-                        img1,
-                        imu_time,
+                        imu_cam_type{
```

Comment:
We can just forward args directly.

Reply:
@mhuzai  So,I had tried this but removing the type I end up with compilation errors

> /home/lincoln/workspace/ILLIXR/realsense/plugin.cpp:130:47: error: no matching member function for call to 'allocate'
                      _m_imu_cam.put(_m_imu_cam.allocate<imu_cam_type>(
                                     ~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
  /home/lincoln/workspace/ILLIXR/realsense/common/switchboard.hpp:471:29: note: candidate function template not viable: no known conversion from 'ILLIXR::time_type' (aka 'time_point<std::chrono::_V2::system_clock, duration<long, ratio<1, 1000000000> > >') to 'ILLIXR::imu_cam_type' for 1st argument
          ptr<specific_event> allocate(Args&&... args) {